lib/users/api.js

```diff
@@ -17,22 +17,10 @@ const auth = require('./auth.js');
 const jobs = require('./jobs.js');
 const requests = require('./requests.js');
 const utils = require('./utils.js');
-
-/**
- * Enum describing the possible types of analytics.
- *
- * @enum {string}
- */
-const AnalyticType = {
```

Comment:
I think this was supposed to be exported (for use by clients, like https://github.com/voxel51/api-py/blob/89f0f20d43dfb0d672cd08847a2f54550a9076b8/voxel51/users/api.py#L37-L41), so it should be re-added and fixed

Reply:
That makes sense

Reply:
yes should have been exported